### ISSUE TITLE
Google Search API broken

### ISSUE THREAD

**Issue by jaraco** [yougov/pmxbot#38]
As an early Christmas gift, Google has disabled their Web Search API. `test_google` now fails in the test suite.

----------------------------------------
- Bitbucket: https://bitbucket.org/yougov/pmxbot/issue/38
- Originally reported by: Jason R. Coombs
- Originally created at: 2014-12-18T23:20:55.993

**Comment by jaraco** [comment]
Catch and report the error. Ref #38.

→ <<cset 3e596fc7d734>>

----------------------------------------
Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Mark test as xfail. Ref #38.

→ <<cset db8267a29961>>

----------------------------------------
Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Fixed in a20684878fe7.

----------------------------------------
Original comment by: Jason R. Coombs

**Comment by jaraco** [comment]
Hmm. It's disabled again.

```
    @command(aliases='g')

    def google(client, event, channel, nick, rest):

    	"Look up a phrase on google"

    	BASE_URL = 'http://ajax.googleapis.com/ajax/services/search/web?'

    	params = dict(

    		v='1.0',

    		q=rest.strip(),

    	)

    	url = BASE_URL + urllib.parse.urlencode(params)

    	resp = requests.get(url)

    	resp.raise_for_status()

    	results = resp.json()

    	if not results['responseData']:

>   		raise RuntimeError(results['responseDetails'])

E     RuntimeError: The Google Web Search API is no longer available. Please migrate to the Google Custom Search API (https://developers.google.com/custom-search/)
```
